src/views/HomePage.js

```diff
@@ -25,6 +25,17 @@ class HomePage extends React.Component {
                 </Jumbotron>
               </Col>
             </Row>
+            <Card style={{ width: '18rem' }}>
+              <Card.Img variant="top" src="https://picsum.photos/id/281/200/175" />
+                <Card.Body>
+                  <Card.Title>City Explorer</Card.Title>
+                    <Card.Text>
```

Comment:
several things in this block that are off on indentation, appearing as if they're nested when they're not